JSON/label.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Lables": [
+    {
+      "title": "The Shawshank Redemption",
+      "color": "red"
+    },
+    {
+      "title": "Title 2",
+      "color": "green"
+    },
+    {
+      "title": "Title 3",
+      "color": "blue"
+    }
+  ]
+}
```

Comment:
As suggested by the previous code reviewerWhen creating a book there is no way to add the label "title and color" which means the label can't be saved in the label.json if can't create it, kindly fix that it says in the project requirements that:
Preserving data

Verify if data is saved after quitting the app:

Open the console app and add a few items.
Close the app.
Open the app again and check it the items you added before are listed after using the correct option


Check if there is a method that saves data in .json files. That method should be used when user quits the app.
Check if data from the .json files is loaded when the app starts.

Make sure that app does not crashes if the .json file does not exist or is empty.